DNN Platform/Modules/DnnExportImport/Components/Services/PagesExportService.cs

```diff
@@ -1015,17 +1017,19 @@ private int ImportModulePermissions(ModuleInfo localModule, ExportModule otherMo
                             local.RoleID = roleId.Value;
                         }
 
-                        other.LocalId = localModule.ModulePermissions.Add(local);
-                        var createdBy = Util.GetUserIdByName(_exportImportJob, other.CreatedByUserID, other.CreatedByUserName);
-                        var modifiedBy = Util.GetUserIdByName(_exportImportJob, other.LastModifiedByUserID, other.LastModifiedByUserName);
-                        UpdateModulePermissionChangers(local.ModulePermissionID, createdBy, modifiedBy);
+                        other.LocalId = localModule.ModulePermissions.Add(local, true);
```

Comment:
Why is it important to add the second parameter?

Reply:
when add the permission into collection, second parameter will make sure the permission is unique, and won't cause index duplicated error when save data into db.

Reply:
to prevent duplicate entries in the database, use a unique constraint (or index) and proper merge statement in add/update stored procedure